src/js/percircle.js

```diff
@@ -123,38 +123,67 @@
                 }, 1000);
             } else if(perdown) {
                 var secs = percircle.attr('data-secs') || options.secs;
-				var timeUpText = percircle.attr('data-timeUpText') || options.timeUpText;
-				if (timeUpText.length > 8) timeUpText='the end';
-                
-				var counter=setInterval(timer, 1000); 
+                var timeUpText = percircle.attr('data-timeUpText') || options.timeUpText;
+                var reset = percircle[0].hasAttribute('data-reset') || options.reset;
```

Comment:
Sorry, just curious, why this one doesn't retrieve the `reset` attribute as it's done in the rest of the cases (i.e. `percircle.attr('data-reset')`